docs/whats-new.rst

```diff
@@ -8,6 +8,11 @@ What's New
 v0.2.1 (unreleased)
 -------------------
 
+Documentation
+~~~~~~~~~~~~~
+
+- corrected link to doc badge via :pull:`213`
```

Comment:
How about "Corrected link to documentation badge on repository main page (:pull:`213`).  By `DaCoEx <https://github.com/dacoex>`_."